gum/arch-arm64/gumarm64writer.c

```diff
@@ -1429,7 +1429,7 @@ gum_arm64_writer_strip (GumArm64Writer * self,
   if (self->ptrauth_support != GUM_PTRAUTH_SUPPORTED)
     return value;
 
-  if (self->target_os == GUM_OS_IOS)
+  if (self->target_os == GUM_OS_IOS || self->target_os == GUM_OS_MACOS)
```

Comment:
```suggestion
  if (self->target_os == GUM_OS_MACOS || self->target_os == GUM_OS_IOS)
```

(Nit: for consistency with ordering elsewhere.)